src/objectTranslations/Address_Verification_Settings__c-de.objectTranslation

```diff
@@ -42,57 +42,57 @@
     </caseValues>
     <fields>
         <help>Für die Adressenüberprüfung verwendeter URL</help>
-        <label><!-- Address Verification Endpoint-DEPRECATED --></label>
+        <label>Endpunkt d. Adressenüberprüf. – ERSETZT</label>
         <name>Address_Verification_Endpoint__c</name>
     </fields>
     <fields>
-        <help>Die Authentifizierungskennung Ihres geheimen Schlüsselpaars</help>
-        <label><!-- Auth ID-DEPRECATED --></label>
+        <help>Die Authentifizierungskennung Ihres geheimen Schlüsselpaars.</help>
+        <label>Authentifizierungskennung – ERSETZT</label>
         <name>Auth_ID__c</name>
     </fields>
     <fields>
-        <help>Der Authentifizierungstoken Ihres geheimen Schlüsselpaars</help>
-        <label><!-- Auth Token-DEPRECATED --></label>
+        <help>Der Authentifizierungstoken Ihres geheimen Schlüsselpaars.</help>
+        <label>Authentifizierungstoken – ERSETZT</label>
         <name>Auth_Token__c</name>
     </fields>
     <fields>
         <help>Bei Aktivierung dieser Option werden geänderte Adressen automatisch mit der Antwort vom Service aktualisiert (falls ein überprüften Ergebnis vorhanden ist).</help>
-        <label><!-- Auto-Update Addresses-DEPRECATED --></label>
+        <label>Adressen autom. aktualisieren – ERSETZT</label>
         <name>Auto_Update_Addresses__c</name>
     </fields>
     <fields>
-        <help>Für die Kontaktierung der SmartlyStreets-API und Überprüfung der Adressen zuständige Salesforce-Klasse</help>
```

Comment:
Shouldn't this be SmartyStreets? I am not sure how to edit directly

Reply:
It is corrected to "smarty" in the new delivery. See green changes on the right side :)